### DISCUSSION TITLE
Replace GRFCrawler with Community NewGRF list when searching external sites

### DISCUSSION THREAD

**Discussion by kevinfields777** [OpenTTD/OpenTTD#9800] [Features]
Currently when you use the in-game content download server and want to search for a file off-site, the "Check External Website" button takes you to https://grfcrawler.tt-forums.net/  This site is no longer being updated, and the SSL certificate is invalid, making the site inaccessible. 

I would like to suggest that it be replaced with the Community NewGRF list at https://wiki.openttd.org/en/Community/NewGRF/ which is regularly updated and is most likely the most reliable list of NewGRFs that are not found via in-game content download.

**Comment by orudge** [comment]
FWIW, I've now fixed the SSL certificate on the site - apologies for that. As far as I'm aware, GRFCrawler should still be functional and you should still be able to add/amend entries on the site.

**Reply by kevinfields777** [reply]
Do we still want to use GRFCrawler? It hasn't been actively promoted in years and Oscar said that he wanted to take it down.

**Reply by orudge** [reply]
I haven't seen the latter remark (and Oskar hasn't contacted me asking to take it down), and as long as it's not a security risk, I'm happy to keep it up.

Keep in mind too that the "search external websites" submits a list of GRFIDs (or textual descriptions) to GRFCrawler to search, rather than just showing a static wiki page. Not that, in my brief look yesterday, I could find any active servers that had GRFs that weren't on BaNaNaS (or on my own PC I guess, should maybe try with an empty OpenTTD folder).

**Comment by LC-Zorg** [comment]
Overall, the wiki page is much more helpful today, but it still doesn't contain all the add-ons currently available, and not even all that are on the GRF Crawler site. Maybe a better solution would be an additional window with a list of pages instead of a direct link?

Anyway, what if all add-ons were listed in the in-game content downloading window? It makes me wonder if it makes sense to keep these pages, since all (almost all) this information could be found in one window in the game.

I redoed my earlier graphic with this window a bit.
![NewGRF - new downloading window v2 1](https://user-images.githubusercontent.com/67662046/151719531-573ea4ab-01aa-4765-b95c-a30b4310337c.png)


Simplified a bit but maybe still overloaded. It could even look significantly different, but the general idea is that the player doesn't have to wander the Internet for hours and look for very basic information, since he could find everything in the game. Some time ago on the Steam forum there was a [discussion](https://steamcommunity.com/app/1536610/discussions/0/5625537320588066619/) about the mess in game and in fact mostly in add-ons. I believe this is very true.
Regarding the wiki page itself, I think it could still be useful, though some changes could be needed. First of all, it would be good for the website to be integrated with the bananas. Currently, you still cannot see e.g. BRIX, City HQ, Chinese HQ, Main HQ, Burj Al Arab HQ and probably many others... I think it is difficult and unreliable to control everything manually.
